go.mod

```diff
@@ -100,5 +100,8 @@ require (
 	willnorris.com/go/imageproxy v0.9.0
 )
 
-// Workaround for https://github.com/golang/go/issues/30831 and fallout.
-replace github.com/golang/lint => github.com/golang/lint v0.0.0-20190227174305-8f45f776aaf1
+replace (
+	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
Does `git.apache.org/thrift.git => github.com/apache/thrift` also work?

Reply:
> Does `git.apache.org/thrift.git => github.com/apache/thrift` also work?

No, it doesn't work. It throws `replacement module without version must be directory path (rooted or starting with ./ or ../)
go: errors parsing go.mod:`

Reply:
Your right. I thought there was an simple way to fix this.